lib/Shared.js

```diff
@@ -32,6 +32,10 @@ module.exports = function Shared(context) {
 					options.filename = new RegExp("^[\/]{0,1}" + str + "$");
 				}
 			}
+			// add MIME type for WebAssembly
```

Comment:
This should ideally be handled by the mime module. I will report that.

Reply:
yeah I'd rather wait on upstream for that to land, implement here as a last resort